pymemcache/test/test_benchmark.py

```diff
@@ -103,7 +102,7 @@ def test_bench_set_multi(request, client, pairs, count):
 
 @pytest.mark.benchmark()
 def test_bench_delete(request, client, pairs, count):
-    benchmark(count, client.delete, six.next(six.iterkeys(pairs)))
+    benchmark(count, client.delete, next(pairs.keys()))
```

Comment:
Could this be just `next(pairs)`?

Reply:
Yes it could be https://six.readthedocs.io/#six.iterkeys

Reply:
I think you could drop the `.keys()` here.